test/functional/rpc_packages.py

```diff
@@ -356,5 +357,33 @@ def test_submitpackage(self):
         assert_equal(res["tx-results"][sec_wtxid]["error"], "version")
         peer.wait_for_broadcast([first_wtxid])
 
+    def test_maxfeerate_maxburn_submitpackage(self):
+        node = self.nodes[0]
+        # clear mempool
+        deterministic_address = node.get_deterministic_priv_key().address
+        self.generatetoaddress(node, 1, deterministic_address)
+
+        self.log.info("Submitpackage maxfeerate arg testing")
+        chained_txns = self.wallet.create_self_transfer_chain(chain_length=2)
+        chain_hex = [t["hex"] for t in chained_txns]
+        pkg_result = node.submitpackage(chain_hex, 0.00000001)
+        assert_equal(pkg_result["tx-results"][chained_txns[0]["wtxid"]]["error"], "max sane feerate exceeded")
+        assert_equal(pkg_result["tx-results"][chained_txns[1]["wtxid"]]["error"], "bad-txns-inputs-missingorspent")
+        assert_equal(node.getrawmempool(), [])
+
+        self.log.info("Submitpackage maxburnamount arg testing")
+        tx = tx_from_hex(chain_hex[1])
+        tx.vout[-1].scriptPubKey = b'a' * 10001 # scriptPubKey bigger than 10k IsUnspendable
+        chain_hex = [chain_hex[0], tx.serialize().hex()]
+        # burn test is run before any package evaluation; nothing makes it in and we get broader exception
+        assert_raises_rpc_error(-25, "Unspendable output exceeds maximum configured by user", node.submitpackage, chain_hex, 0, 0.1)
+        assert_equal(node.getrawmempool(), [])
+
+        # Turn off restrictions for both and send it; parent gets through as own subpackage
```

Comment:
To turn off for maxfeerate you should pass 0 fee rate, you cant turn off the restriction of `maxburnamount`
nit: maybe reword to relax?
```suggestion
# Relax the restrictions and send it; parent gets through as own subpackage
```

Reply:
taken